packages/yoshi-flow-editor/src/model.ts

```diff
@@ -208,8 +211,8 @@ For more info, visit http://tiny.cc/dev-center-registration`);
 
   const model = {
     appName: config.name,
-    sentry: appConfig && appConfig.sentry ? appConfig.sentry : null,
-    appDefId: appConfig ? appConfig.appDefinitionId : null,
+    sentry: (shouldUseSentry() && appConfig.sentry) || null,
+    appDefId: appConfig.appDefinitionId ?? null,
```

Comment:
if prev condition is valid, then:
```suggestion
    appDefId: appConfig?.appDefinitionId ?? null,
```

Reply:
Ah, you changed warning to `throw new Error(`.
Okok

Reply:
yeah, I think since we are providing wizard that generates it, we don't need to hande cases w/o config